### DISCUSSION TITLE
Using climate entity without relays

### DISCUSSION THREAD

**Discussion by jacauc** [Blackymas/NSPanel_HA_Blueprint#2273] [Q&A] [answered]
Hi,

I have an external heater which is powered by a Shelly switch. I would like to have the climate addon control when to switch on/off the heater switch. 

Switch shelly on when actual indoor temperature is below target temperature
Switch shelly off when actual indoor temperature is above target temperature

The heater unit and shelly switch is on the opposite side of the room, so I am just looking for a way to make the NSPanel act as a relatively dumb thermostat.

Do I need to do this with an external automation or is it just something I am missing in the automation settings?

Thanks in advance, this blueprint is incredible!

**Comment by edwardtfn** [comment, marked as answer]
There are multiple ways of doing this, but as you will be dependant on the wifi to control the Shelly, I believe the best would be setting up a "[Generic Thermostat](https://www.home-assistant.io/integrations/generic_thermostat/)" in Home Assistant, pointing to your Shelly and the heater and any temperature sensor (including the one from your panel) as the target sensor.
That will create a climate entity which you will be able to assign to most of the buttons in your panel.

**Reply by jacauc** [reply]
Thank you! So easy!
